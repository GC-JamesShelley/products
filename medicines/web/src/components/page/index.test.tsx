import { shallow } from 'enzyme';
import React from 'react';
import App from './index';

describe(App, () => {
  it('should render', () => {
    // tslint:disable-next-line: no-empty
    const noop = () => {};
    const component = shallow(
      <App
        children={<></>}
        title={'MHRA'}
        storageAllowed
        setStorageAllowed={noop}
      />,
    );
    expect(component).toMatchSnapshot();
  });
});
