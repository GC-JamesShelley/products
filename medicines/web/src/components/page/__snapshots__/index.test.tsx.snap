// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<Fragment>
  <Head>
    <title>
      MHRA 
      MHRA
    </title>
    <meta
      content="base-uri 'self'; default-src 'self'; script-src 'self'; style-src 'self'; object-src 'none'; form-action 'self'; font-src 'self'; connect-src 'self'; img-src 'self';"
      httpEquiv="Content-Security-Policy-Report-Only"
    />
  </Head>
  <styled.div>
    <GlobalStyleComponent />
    <CookieBanner
      setStorageAllowed={[Function]}
      storageAllowed={true}
    />
    <header
      title="MHRA"
    />
    <styled.main />
    <Footer />
  </styled.div>
</Fragment>
`;
